sktime/transformations/series/bootstrapping.py

```diff
@@ -0,0 +1,401 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Bootstrapping methods for time series."""
+
+__author__ = ["ltsaprounis"]
+
+from copy import copy
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.api import STL as _STL
+
+from sktime.transformations.base import BaseTransformer
+from sktime.transformations.series.boxcox import BoxCoxTransformer
+
+
+class BootsrappingTransformer(BaseTransformer):
+    """Creates a population of similar time series.
+
+    This method utilises a form of bootstrapping to generate a population of
+    similar time series to the input time series [1]_, [2]_.
```

Comment:
the references are textbooks, can you kindly point to the section/chapter this is specifically implementing?